### PR TITLE
Partner dashboard

### PR DESCRIPTION
**Pull request by saloni-080601**
**Which issue does this refer to ?**
A clear and concise description. Use `#` to refer to github issues and PR

**Brief description of the changes done**
A brief step by step changes made. Example :
1. Upgraded packages
2. Added extra state to handle clicks
3. ...

**People to loop in / review from**
Use `@` key to tag people to review from

### CODE REVIEW

File: src/pages/NewPartner/index.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+                            {`${partner[item].City} , ${partner[item].State}`}
+                          </Typography>
+                        </CardContent>
+                        <CardActions sx={{ height: "10px" }}>
+                          {partner[item].Url !== null ? (
+                            <IconButton>
+                              <Link href={partner[item].Url} target="_blank">
+                                <PublicIcon variant="outlined" />
+                              </Link>
+                            </IconButton>
```
Comment (Poonam-Singh-Bagh) [inline src/pages/NewPartner/index.js]: Icons are not directing to any link

Comment (Poonam-Singh-Bagh) [inline src/pages/NewPartner/index.js]: Aslo use `<ExternalLink href={partner[item].Url}><PublicIcon variant='outlined' /></ExternalLink>` instead

**Comment by Poonam-Singh-Bagh** [comment]
Screenshot from 2022-06-07 13-24-56

Icons should be at the bottom even if card content is less then other cards

**Comment by Poonam-Singh-Bagh** [comment]
Screenshot from 2022-06-07 13-25-30

Screenshot from 2022-06-07 13-25-40

Add route to the Header and Footer as `Our Partner`

File: src/constant.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ export const PATHS = {
   CLASS: "/class",
   COURSE: "/course",
   PATHWAY_COURSE_CONTENT: "/course-content/:pathwayId/:courseId/:exerciseId",
+  PATHWAY_COURSE_CONTENT: "/course-content/:pathwayId/:courseId/:exerciseId",
   COURSE_CONTENT: "/course/:courseId",
```
Comment (Poonam-Singh-Bagh) [inline src/constant.js]: Why did you add line number 9?

File: src/components/Dashboard/StudentData/index.js [hunk 3]
```diff
@@ -367,15 +369,18 @@ function StudentData(props) {
...
           <button
             onClick={() => {
               setFilteredData(false);
               setFilterVal([0, 0]);
             }}
             className="filter-clear"
+            disabled={isDisabled}
           >
             clear
           </button>
```
Comment (Poonam-Singh-Bagh) [inline src/components/Dashboard/StudentData/index.js]: Make `isDisable` true when click on clear button

File: src/constant.js [hunk 4]
```diff
@@ -31,8 +31,8 @@ export const PATHS = {
   OUR_PARTNER: "/our-partner",
   OUR_STORY: "/our-story",
   SEARCHED_COURSE: "/search-course",
+  NEW_PARTNER: "/new-partner",
```
Comment (Poonam-Singh-Bagh) [inline src/constant.js]: Delete `OurPartner` component and named you component as `OurPartner` and so route. and then you can delete `NEW_PARTNER: "/new-partner",`

File: src/pages/Login/index.js [hunk 5]
```diff
@@ -111,7 +111,7 @@ function Login(props) {
       <Container className={classes.merakiLogin} maxWidth="lg">
         <Grid container spacing={2}>
           <Grid item xs={12} ms={6} md={6}>
-            <Container maxWidth="sm">
+            <Container maxWidth="md">
               <Typography
```
Comment (Poonam-Singh-Bagh) [inline src/pages/Login/index.js:123]: Is there any specific reason to change the container size from sm to md?

File: src/routing/index.js [hunk 6]
```diff
@@ -26,7 +26,7 @@ import StatePartnerDashboard from "../pages/StatePartner Dashboard";
 import OurStory from "../pages/OurStory";
 import Team from "../pages/Team";
 import Partner from "../pages/OurPartner";
-
+import NewPartner from "../pages/NewPartner";
```
Comment (Poonam-Singh-Bagh) [inline src/routing/index.js]: Change the name from `import NewPartner from "../pages/NewPartner";` to  `import OurPartner from "../pages/OurPartner";` after renamed you component and delete previous OurPartner component

File: src/routing/index.js [hunk 7]
```diff
@@ -71,6 +71,7 @@ const Routing = () => {
         path={PATHS.NAVGURUKUL_INTRODUCE}
         component={NavgurukulIntroduce}
       />
+      <Route exact path={PATHS.NEW_PARTNER} component={NewPartner} />
```
Comment (Poonam-Singh-Bagh) [inline src/routing/index.js]: Same as upper comment from `<Route exact path={PATHS.NEW_PARTNER} component={NewPartner} />` to <Route exact path={PATHS.OUR_PARTNER} component={OurPartner} />

**Review by Poonam-Singh-Bagh — CHANGES_REQUESTED** [review]

**Review by komalahire — DISMISSED** [review]

**Pull request merged**